srcalc/src/test/java/gov/va/med/srcalc/web/view/admin/EditRiskModelValidatorTest.java

```diff
@@ -0,0 +1,51 @@
+package gov.va.med.srcalc.web.view.admin;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import gov.va.med.srcalc.util.ValidationCodes;
+
+import org.junit.Test;
+import org.springframework.validation.BeanPropertyBindingResult;
+import org.springframework.validation.BindingResult;
+
+public class EditRiskModelValidatorTest {
```

Comment:
The formatting is inconsistent in this class. Can you have Eclipse reformat the file?